pkgs/test_api/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.2.15-dev
+
+* Cancel any StreamQueue that is created as a part of a stream matcher once it
```

Comment:
[nit] Can we copy this into the test changelog? That's the one most users will see.

Reply:
hmm, its a bit complicated since I wasn't planning on publishing this yet? so putting it in the test changelog might not be accurate (we already have one pending change there which could get published before this does)

Reply:
Fair enough. We can consider it a part of the process to rectify changelogs in the commits where we bump the dependency.

Reply:
Ya I think that would be the best time to do it - not sure the right way to codify that process though.